src/data/ConstantD2Repository.ts

```diff
@@ -12,6 +12,12 @@ import { writeFileSync } from "fs";
 export class ConstantD2Repository implements ConstantRepository {
     constructor(private api: D2Api) {}
 
+    async get(): Promise<string[]> {
+        const { constants } = await this.api.metadata.get({ constants: { fields: { code: true } } }).getData();
+
+        return constants.filter(constant => constant.code).map(constant => constant.code);
```

Comment:
Right now this implementation works for the `GetAutogenConfigSchemaUseCase` usecase because in there we need all the codes. But if in the future we need an array of constants (not just the code or any specific attribute) in other usecase we'll need to implement a new method. 

So my proposal here is to return an array of Constants[] and then in the usecase add the logic to only get the codes. Making our implementation more generic and easy to use for any future usecase. Does it make sense?